notes/how-to-read-a-paper.md

```diff
@@ -54,11 +54,11 @@ The third pass can take multiple hours even for experienced readers.
 This is also known as a literature review. A [search](https://www.quora.com/What-is-a-literature-survey-in-any-project-report) returns that it is a study 
 of existing material on the same (or similar) topic of the paper you're reviewing:
 1. Existing theories about the topic which are accepted universally.
-2. Books written on the topic, both generic and specific.
+2. Books are written on the topic, both generic and specific.
```

Comment:
This is the only change that I think doesn't make sense. These numbered items are a list of things that would be included in a literature review, each item doesn't necessarily have to be a full sentence.

Reply:
Fixed